Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -347,7 +279,88 @@ - (void)searchBarCancelButtonClicked:(UISearchBar *)SearchBar {
 
 - (IBAction)didTapDays:(id)sender {
 }
+
 - (IBAction)goToToday:(id)sender {
     [self goToTodayAnimated:YES];
 }
+
+#pragma mark - Utility Methods
+- (void)showErrorAlert:(NSError *)error {
```

Comment:
Yeah, I think the only reason we have two separate error messages is for developers to differentiate which issue is going wrong with the app, which shouldn't really be something being shown to endusers anyway. I agree with you on consolidation.